controllers/employeesController.js

```diff
@@ -0,0 +1,142 @@
+const { type } = require('express/lib/response');
+const { check } = require('prettier');
+const db = require('../models');
+
+function getAttributesFromRequest(req){
+	return {
+		name: req.body.name,
+		lastName: req.body.lastName,
+		position: req.body.position
+	}
+}
+function validateEmployee(req, res) {
+	if (attributesInRequest(req, res) && attributesInRequestDefined(req, res) && checkTypeOfAttributes(req, res)) {
+		return true;
+	}
+	return false;
+}
+function attributesInRequest(req, res) {
+	if (emptyObject(req.body)) {
+		res.status(400).send({
+			message: 'No attributes in body',
+		});
+		return false;
+	}
+	return true;
+}
+
+function attributesInRequestDefined(req, res) {
+	let {name: name, lastName: lastName, position: position} = getAttributesFromRequest(req);
+
+	if (name == undefined || lastName == undefined || position == undefined) {
+		res.status(400).send({
+			message: 'No name / last name / position filled',
+		});
+		return false;
+	}
+	return true;
+}
+
+function checkTypeOfAttributes(req, res) {
+	let {name: name, lastName: lastName, position: position} = getAttributesFromRequest(req);
```

Comment:
Same as line 29

Reply:
Resolved in 83d7de4ed11bca24c689d1fe577bc38c8e589539